client/web/src/enterprise/batches/list/GettingStarted.tsx

```diff
@@ -149,7 +149,13 @@ export const GettingStarted: React.FunctionComponent<GettingStartedProps> = ({ f
                             View the product marketing page for a high-level overview of benefits and customer use
                             cases.
                         </p>
-                        <Link to="https://about.sourcegraph.com/batch-changes" rel="noopener">
+                        {/*
+                            a11y-ignore
+                            Rule: "color-contrast" (Elements must have sufficient color contrast)
+                            There are a lot of `Link` usages in `--color-bg-2=--gray-03` container, to fix this issue
+                            we can use `--gray-02` or find another color for `--link-color`
```

Comment:
```suggestion
                            There are a lot of `Link` usages in `--color-bg-2=--gray-03` container, to fix this issue
                            we can use `--gray-02` or find another color for `--link-color`
```

@gitstart-sourcegraph Lets just link to a GitHub issue for these contrast issues. I think we will encounter quite a few of them. We might not want to address all of these as the WCAG color contrast algorithm is changing to be more accurate in the next version of the specification (WCAG 3). Let's leave a comment with a link to this issue wherever we have this problem (and it doesn't appear to be a genuine bug) .

```suggestion
                            GitHub issue: https://github.com/sourcegraph/sourcegraph/issues/33343
```